cv32/sim/uvmt_cv32/vcs.mk

```diff
@@ -47,7 +47,8 @@ VCS_COMP_FLAGS  ?= -lca -sverilog -top uvmt_cv32_tb \
 										-assert svaext -race=all -ignore unique_checks -full64
 VCS_GUI         ?=
 VCS_RUN_COV      = -cm line+cond+tgl+fsm+branch+assert -cm_dir $(MAKECMDGOALS).vdb
-NUM_TEST         ?= 2
+NUM_TESTS         ?= 2
+XRUN_UVM_VERBOSITY ?= UVM_MEDIUM
```

Comment:
Are you using **XRUN**_UVM_VERBOSITY deliberately?  Perhaps you intended to use **VCS**_UVM_VERBOSITY?

Reply:
That was a copy-paste error